### PR TITLE
Remove leftover references to `pkg`

### PR DESCRIPTION
**Pull request by TooTallNate**
We are no longer compiling `now-cli` with `pkg` so this is dead code.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -43,16 +43,10 @@ const NOW_CONFIG_PATH = configFiles.getConfigFilePath();
...
 
 epipebomb();
 
-// we only enable source maps while developing, since
-// they have a small performance hit. for this, we
-// look for `pkg`, which is only present in the final bin
-if (!insidePkg) {
-  sourceMap.install();
-}
+sourceMap.install();
```
Comment (AndyBitz) [inline src/index.js:49]: But now they are always installed instead of just during development, right?

Comment (TooTallNate) [reply src/index.js:49]: That was already the case once we stopped using `pkg`. Personally I'd prefer stack traces enabled for Sentry reports.

Comment (TooTallNate) [reply src/index.js:49]: I also can't think if a decent "is development" check anymore 🤔

Comment (AndyBitz) [reply src/index.js:49]: Agreed, maybe through `--debug` or an env var, but since it was the case already anyway I think it's good to go

**Review by AndyBitz — APPROVED** [review]

**Pull request merged**